source/includes/_tipser-elements-analytics.md

```diff
@@ -1,189 +1,308 @@
----------        | -----------    | -----------       
-`E-commerce`     | `remove`       | Yes (with one product attached)
-
-## Checkout
-
-This event is emitted when checkout dialog is opened with a product (or several products).
-
-[![](product_checkout.png)](/images/product_checkout.png)
-
-_In the screenshot above checkout dialog has been displayed and "Checkout" event has been emitted for a product "Retexturising SERUM"_
-
-### Additonal notes ###
-
-Please note that "checkout" is not the same thing as "purchase". The event is emitted even if the product checkout is never finalized.
-The event is emitted every time checkout dialog is opened. This means that there may be several "Checkout" events emitted for product that was purchased just 
-once or that was never purchased.
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?     
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
-
-
-## Purchase
-
-This event is emitted when a product (or several products) is purchased.
-
-[![](purchase_event.png)](/images/purchase_event.png)
-
-_In the screenshot above the user has finalized the payment and the order confirmation dialog has been displayed. "Purchase" event has been emitted._
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?        
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
+# Tipser Elements Analytics
+
+Tipser has implemented an event for the Tipser Elements Analytics that users of the script can listen to in order to gather analytics about Tipser interactions and feed them into their own analytics engines.
+
+## General overview of the Tipser Elements Analytics collector
+By listening to the event `tw-track` certain Tipser events can be gathered and pushed into an external analytics engine.
+
+The event contains the object detail that contains the following items:
+
+`detail` object
+
+```yaml
+{
+  action: string, # what action is taken on the target, mandatory
+  target: string, # where this action is taken, mandatory
+  description: string, # describing what is being logged
+  object: # details about the tracked object, e.g. collection ID, product ID, etc.
+  [    
+    {  
+      # some properties here               
+    },     
+    ..., # this array can contain more than 1 object
+  ]
+}
+```
+
+
+Example of an event listener:
+
+```javascript
+document.addEventListener('tw-track', function (e) {
+  var detial = e.detail; 
+  console.log('tw-track');
+  console.log('tw-track action',detial.action);
+  console.log('tw-track description',detial.description);
+  console.log('tw-track target',detial.target);
+  console.log('tw-track object',detial.object);
+});
+```
+
+## List of supported events
```

Comment:
I will replace "List of supported events" with "List of supported interactions". Because technically event is only one `tw-track`, but called with different data for each interactions.

Reply:
I'm not sure. "Interactions" sounds a bit vague to me. There are many more possible interactions with Tipser Elements for which we don't send any analytics events, so I think the term "interaction" is too broad. I think "events", even if it may be not 100% technically correct, is clear enough.

Reply:
> Tipser has implemented an event for the Tipser Elements Analytics that users of the script can listen to in order to gather analytics about Tipser interactions and feed them into their own analytics engines.

- ^^ There is a singular word of `event` used, and plural `interactions`. So this is what it is already. What I did it is more consistent with this main description. We have one "event", and many and many "interactions". 
- "Event" is as broad term as "interaction". So your argument does work for me :)
- User does not care whether something is called Event or Interactions, IMO both will work for him. But this is just inconsistent form technical point of view, and it could lead confusion. IMO bigger then "interactions" word.